moveit_experimental/moveit_jog_arm/src/low_pass_filter.cpp

```diff
@@ -59,19 +59,22 @@ LowPassFilter::LowPassFilter(double low_pass_filter_coeff)
   // guarantee this doesn't change because the logic below depends on this length implicity
   static_assert(LowPassFilter::FILTER_LENGTH == 2, "moveit_jog_arm::LowPassFilter::FILTER_LENGTH should be 2");
 
-  ROS_ASSERT_MSG(!std::isinf(feedback_term_), "%s: outputs from filter will be inf because feedback term is inf",
-                 LOGNAME);
-  ROS_ASSERT_MSG(!std::isinf(scale_term_), "%s: outputs from filter will be inf because denominator of scale is 0",
-                 LOGNAME);
-  ROS_ASSERT_MSG(low_pass_filter_coeff >= 1., "%s: Filter coefficient < 1. makes the lowpass filter unstable", LOGNAME);
+  if (low_pass_filter_coeff < 1)
+  {
+    ROS_WARN_STREAM_NAMED(LOGNAME, "Filter coefficient < 1. makes the lowpass filter unstable. "
+                                   "Setting to default of 10.");
+    low_pass_filter_coeff = 10;
+    scale_term_ = 1. / (1. + low_pass_filter_coeff);
+    feedback_term_ = 1. - low_pass_filter_coeff;
+  }
```

Comment:
[nit] I like the softer error handling here, however I suggest making that an error instead of warn log to make it clearer to the users that the parameter that was set didn't take.